setup.py

```diff
@@ -31,13 +31,13 @@
     'url': '',
     'license': 'BSD 3-Clause',
     'install_requires': [
-        'pynwb>=1.1.2'
+        'pynwb>=1.3.0'
     ],
     'packages': find_packages('src/pynwb'),
     'package_dir': {'': 'src/pynwb'},
     'package_data': {'ndx_mies': [
-        'spec/ndx-MIES.namespace.yaml',
-        'spec/ndx-MIES.extensions.yaml',
+        'spec/ndx-mies.namespace.yaml',
+        'spec/ndx-mies.extensions.yaml',
```

Comment:
it probably doesn't matter, but these paths have lowercase "mies" but the names of the files in the repo are still uppercase "MIES"

Reply:
Thanks, fixed.